src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -22,16 +22,23 @@ import { MatrixClient } from "../client";
 import { EventType } from "../@types/event";
 import { CallMembership, CallMembershipData } from "./CallMembership";
 import { Focus } from "./focus";
+import { randomString } from "../randomstring";
+import { MatrixEvent } from "../matrix";
+import { EncryptionKeyEventContent } from "./types";
 
 const MEMBERSHIP_EXPIRY_TIME = 60 * 60 * 1000;
 const MEMBER_EVENT_CHECK_PERIOD = 2 * 60 * 1000; // How often we check to see if we need to re-send our member event
 const CALL_MEMBER_EVENT_RETRY_DELAY_MIN = 3000;
 
+const getNewEncryptionKey = (): string => randomString(32);
```

Comment:
Security horn! 🚨 This used Math.random and is not safe for a security sensitive string! Use `crypto.getRandomValues()`.

Reply:
Good catch - thank you!